backend/src/zac_lite/conf/base.py

```diff
@@ -341,6 +341,11 @@
 ENVIRONMENT = config("ENVIRONMENT", "")
 SHOW_ALERT = True
 
+#
+# EXECUTE TASK TOKEN INVALIDATION
+#
+EXECUTE_TASK_TOKEN_TIMEOUT_DAYS = config("EXECUTE_TASK_TOKEN_TIMEOUT_DAYS", default=7)
```

Comment:
Why 7 days? Is that going to be our cleaning frequency? Would it be desirable to have different timeout durations for different tasks?

Reply:
Yeah, 7 days seems now a reasonable action time but it's just a default now that we can easily customize through the environment with input from FB.

There is a TODO in the token generator to indeed be able to customize the timeout duration per task, but then we would need to start storing task data + timeouts in the database which might not be desirable. For an initial pass, I think a week should suffice.